recipes/imagl/all/conanfile.py

```diff
@@ -1,7 +1,12 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, patch, copy
+from conan.errors import ConanInvalidConfiguration
+import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.59"
 
 
 class ImaglConan(ConanFile):
```

Comment:
Both are OK. https://gitlab-lepuy.iut.uca.fr/opengl/imagl is mirrored to https://github.com/Woazim/imaGL.
I think that people are more used to github than to my own GitLab instance.

Reply:
@Woazim in case https://github.com/Woazim/imaGL is a mirror, please, revert the conandata.yml. Mirror are welcome, but not as main link and they should keep same checksum: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#mirrors